composer.json

```diff
@@ -21,18 +21,18 @@
     "extra": {
     },
     "require": {
-        "php": "^7.3 || ~8.0.0",
+        "php": "^7.3 || ~8.0.0 || ~8.1.0",
         "ext-json": "*",
-        "laminas/laminas-http": "^2.14",
-        "laminas/laminas-json": "^3.2",
-        "laminas/laminas-stdlib": "^3.3",
-        "laminas/laminas-zendframework-bridge": "^1.1"
+        "laminas/laminas-http": "^2.15",
+        "laminas/laminas-json": "^3.3",
+        "laminas/laminas-stdlib": "^3.6",
+        "laminas/laminas-zendframework-bridge": "^1.4"
```

Comment:
The bridge / custom autoloader can be removed here to avoid [performance problems](https://github.com/laminas/laminas-zendframework-bridge/issues/83). This includes another change:

```json
"replace": {
    "zendframework/zendservice-recaptcha": "^3.2.0"
}
```
to:
```json
"conflict": {
    "zendframework/zendservice-recaptcha": "*"
}
```

Reply:
I'd like to provide this in a seperate PR as it is not really related to the PHP 8.1 support.

Reply:
Anyway, but it would be good if we could include it to the [3.4.0 version](https://github.com/laminas/laminas-recaptcha/milestone/6).

Reply:
https://github.com/laminas/laminas-recaptcha/pull/17